fem/bilininteg.hpp

```diff
@@ -14,18 +14,29 @@
 
 #include "../config/config.hpp"
 #include "nonlininteg.hpp"
+#include "fespace.hpp"
+#include "bilininteg_ext.hpp"
 
 namespace mfem
 {
 
 /// Abstract base class BilinearFormIntegrator
 class BilinearFormIntegrator : public NonlinearFormIntegrator
 {
-protected:
+public:
    BilinearFormIntegrator(const IntegrationRule *ir = NULL) :
       NonlinearFormIntegrator(ir) { }
 
 public:
+   /// Method defining partial assembly.
+   virtual void Assemble(const FiniteElementSpace&);
+
+   /// Method for partially assembled action.
+   virtual void MultAssembled(Vector&, Vector&);
```

Comment:
Is there a reason why `MultAssembled` doesn't take a `const Vector&` for first argument? Same for `MultAssembledTranspose`?